metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -476,6 +477,20 @@ object ServerCommands {
     """|This command should be sent in with the LSP [`TextDocumentPositionParams`](https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocumentPositionParams)
        |""".stripMargin,
   )
+  final case class ExtractMethodParams(
+      param: TextDocumentIdentifier,
+      range: lsp4j.Range, // from LSP
+      extractPosition: lsp4j.Range,
+  )
+  val ExtractMethod = new ParametrizedCommand[ExtractMethodParams](
+    "extract-method",
+    "Extract method from range",
+    """|Whenever a user chooses code action to extract method, this command is later ran to
+       |calculate parameters for the new method with types.
+       |""".stripMargin,
+    """|This command should be sent in with the LSP [`TextDocumentPositionParams`](https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocumentPositionParams) and `applRange` Integer
```

Comment:
We should include information with `ExtractMethodParams` showed as a json like for example `DiscoverTestSuites`

Reply:
Could you explain the parameters of `ExtractMethodParams` in the docs?